src/System.Security.Cryptography.X509Certificates/tests/X509StoreTests.cs

```diff
@@ -283,29 +283,13 @@ public static void AddReadOnlyThrowsWhenCertificateExists()
         }
 
         [Fact]
-        public static void RemoveReadOnlyThrowsWhenFound()
```

Comment:
Why are you removing this test?

Reply:
Ah, sorry, I misread the original issue (#31050). I thought it suggested removing the test since it was useless, but I guess it can stay. It was passing anyway with my changes, so it shouldn't be a problem.